lib/content-services/src/lib/search/components/search-datetime-range/search-datetime-range.component.ts

```diff
@@ -133,8 +133,9 @@ export class SearchDatetimeRangeComponent implements SearchWidget, OnInit, OnDes
         if (isValid && this.id && this.context && this.settings && this.settings.field) {
```

Comment:
isValid parameter is conflicting with the method from date-fns. Please rename the parameter name.

Reply:
Added the rename the parameter name.